test/Transforms/RewriteStatepointsForGC/base-pointers-12.ll

```diff
@@ -0,0 +1,93 @@
+; RUN: opt %s -rewrite-statepoints-for-gc -spp-print-base-pointers -S 2>&1 | FileCheck %s
+
```

Comment:
I don't see any checks , so does the `| FileCheck%s` do anything? Or am I overlooking something here?